src/components/filters/Filters.tsx

```diff
@@ -1,129 +1,119 @@
-import { countries } from "@/data/countries";
-import { positions } from "@/data/positions";
-import { tags } from "@/data/tags";
-import FilterTitle from "../filter-title/FilterTitle";
-import { Checkbox } from "../form/checkbox/Checkbox";
-import { Input } from "../form/input/Input";
-import { Tag } from "../form/tag/Tag";
+import { countries } from '@/data/countries';
+import { positions } from '@/data/positions';
+import { tags } from '@/data/tags';
 
-export type categories =
-  | "search"
-  | "country"
-  | "position"
-  | "tag"
-  | "withAudio";
+import FilterTitle from '../filter-title/FilterTitle';
+import { Checkbox } from '../form/checkbox/Checkbox';
+import { Input } from '../form/input/Input';
+import { Tag } from '../form/tag/Tag';
+
+export type categories = 'search' | 'country' | 'position' | 'tag' | 'withAudio';
 
 interface FiltersProps {
-  filters: {
-    search: string;
-    countries: string[];
-    positions: string[];
-    tags: string[];
-    withAudio: boolean;
-  };
-  onChange: (category: categories, key?: string) => void;
-  onClear: (category: categories) => void;
+    filters: {
+        search: string;
+        countries: string[];
+        positions: string[];
+        tags: string[];
+        withAudio: boolean;
+    };
+    onChange: (category: categories, key?: string) => void;
+    onClear: (category: categories) => void;
 }
 
 export const Filters = ({ filters, onChange, onClear }: FiltersProps) => {
-  return (
-    <aside className="flex flex-col justify-between border-b sm:border-r sm:border-b-0 p-5 sm:min-h-full bg-gray-50 pb-10">
-      <section className="flex flex-col gap-6">
-        <div>
-          <FilterTitle
-            title="Search by name"
-            onClear={() => onClear("search")}
-            showClear={!!filters.search}
-          />
-          <Input
-            value={filters.search}
-            onChange={(value) => onChange("search", value)}
-          />
-        </div>
-        <div>
-          <FilterTitle
-            title="Country"
-            onClear={() => onClear("country")}
-            showClear={filters.countries.length > 0}
-          />
-          <div className="flex flex-col gap-1">
-            {countries.map(({ code, label, flag }) => {
-              return (
-                <Checkbox
-                  label={`${flag} ${label}`}
-                  key={code}
-                  value={code}
-                  checked={filters.countries.includes(code)}
-                  onChange={() => onChange("country", code)}
-                />
-              );
-            })}
-          </div>
-        </div>
+    return (
+        <aside className="flex flex-col fixed justify-between border-b overflow-scroll h-full sm:border-r sm:border-b-0 p-5 sm:min-h-full bg-gray-50 pb-10">
```

Comment:
A few notes here:
- This PR is mixing two very different things: Adding configs and changing styles and it's not a good practice. This should be separated
- The padding on the list is broken
- The responsive is broken

<img width="629" alt="Screenshot 2023-03-20 at 08 51 03" src="https://user-images.githubusercontent.com/2673262/226278294-bcbe4447-c3fa-4db4-bcf5-e4c051361734.png">